src/main/resources/templates/layout/base.html

```diff
@@ -2,134 +2,137 @@
                             <a class="govuk-footer__link"
                                href="https://www.nationalarchives.gov.uk/doc/open-government-licence/version/3/"
                                rel="license">Open Government Licence v3.0</a>, except where otherwise stated
                         </span>
-                    </div>
-                    <div class="govuk-footer__meta-item">
-                        <a class="govuk-footer__link govuk-footer__copyright-logo"
-                           href="https://www.nationalarchives.gov.uk/information-management/re-using-public-sector-information/uk-government-licensing-framework/crown-copyright/">
-                            © Crown copyright
-                        </a>
-                    </div>
-                </div>
-            </div>
-        </footer>
-
-        <script src="/govuk-frontend-3.9.1.min.js"></script>
-        <script>
-        window.GOVUKFrontend.initAll()
-
-        </script>
-
-        <script>
-          function checkAndShowAppInfo(message){
-            if (!message) {
-              document.getElementById("application-info").style.display = 'none'
-              document.getElementById("application-message").textContent = ''
-              return
-            }
-
-            document.getElementById("application-message").textContent = message
-            document.getElementById("application-info").style.display = 'block'
-          }
-
-          const appInfoRequest = new XMLHttpRequest()
-
-          appInfoRequest.addEventListener('load', function () {
-            checkAndShowAppInfo(JSON.parse(this.responseText)['message'])
-          })
-
-          appInfoRequest.open('GET', '/app/info');
-          appInfoRequest.send();
-        </script>
-    </body>
-</html>
\ No newline at end of file
+      </div>
+      <div class="govuk-footer__meta-item">
+        <a class="govuk-footer__link govuk-footer__copyright-logo"
+           href="https://www.nationalarchives.gov.uk/information-management/re-using-public-sector-information/uk-government-licensing-framework/crown-copyright/">
+          © Crown copyright
+        </a>
+      </div>
+    </div>
+  </div>
+</footer>
+
+<script src="/govuk-frontend-3.9.1.min.js"></script>
+<script>
+  window.GOVUKFrontend.initAll()
+
+</script>
+
+<script>
+  function checkAndShowAppInfo(message){
+    if (!message) {
+      document.getElementById("application-info").style.display = 'none'
+      document.getElementById("application-message").textContent = ''
+      return
+    }
+
+    document.getElementById("application-message").textContent = message
+    document.getElementById("application-info").style.display = 'block'
+  }
+
+  const appInfoRequest = new XMLHttpRequest()
+
+  appInfoRequest.addEventListener('load', function () {
+    checkAndShowAppInfo(JSON.parse(this.responseText)['message'])
+  })
+
+  appInfoRequest.open('GET', '/app/info');
+  appInfoRequest.send();
+</script>
+</body>
+</html>z
```

Comment:
```suggestion
</html>
```

I think this might be a typo!